### PR TITLE
Menu item as a clickable link

### PR DESCRIPTION
**Pull request by chloe0592**
In previous work we disabled clicking on the menu item parent when it contained a submenu. 

This could lead to users reporting it as a bug, since it's not straightforward why some menu items are clickable and some not. 
But more importantly custom apps  can have different links from the parent menu item and the first submenu child, so it's a potentially a breaking change.

Therefore, we decided to have all menu items as links and clickable.

### CODE REVIEW

**Review by emmenko — APPROVED** [review]
Thanks!

> But more importantly custom apps can have different links from the parent menu item and the first submenu child, so it's a potentially a breaking change.

Actually this is a bad behavior of our internal apps to have a duplicate link for the main item (for example `Product list`).
We discourage this kind of pattern (see note in the docs: https://docs.commercetools.com/custom-applications/api-reference/application-config#submenulinksuripath) and should eventually fix it also in our internal apps.

**Comment by chloe0592** [comment]
> Actually this is a bad behavior of our internal apps to have a duplicate link for the main item (for example `Product list`). We discourage this kind of pattern (see note in the docs: https://docs.commercetools.com/custom-applications/api-reference/application-config#submenulinksuripath) and should eventually fix it also in our internal apps.

Oh, okay, thanks! That's really good to know and then it's certain we need to have all menu items as links regardless if being internal or custom apps.

**Review by kark — APPROVED** [review]
🎉

File: .changeset/big-mayflies-bake.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'@commercetools-frontend/application-shell': patch
+---
+
+Navigation menu item is again clickable even if it has submenu.
```
Comment (CarlosCortizasCT) [inline .changeset/big-mayflies-bake.md:5]: **nit**:
```suggestion
First level navigation menu items are clickable again even it they have submenu items.
```

**Review by CarlosCortizasCT — APPROVED** [review]
Looks good to me.

Thanks! 👍

**Pull request merged**